.github/workflows/push.yml

```diff
@@ -0,0 +1,22 @@
+on:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron:  '0 0 * * *'
```

Comment:
I would like the code to run any time there is a new commit or push of code up from my local computer

Reply:
Every push to the main branch triggers the built. Additionally it is triggered at 00:00.

Reply:
Ah, okay. I had seen the comment that the code runs manually and made a broad assumption :-)

Reply:
Looks like your comments point to everything being okay, so I merged the code and will test.

Reply:
I have created another pull request as the target repository has been mine and not yours. Additionally I added some comments.